geemap/common.py

```diff
@@ -15458,3 +15458,40 @@ def close_btn_click(change):
 
     else:
         return toolbar_widget
+
+
+def geotiff_to_image(image: str, output: str) -> None:
+    """
+    Converts a GeoTIFF file to a JPEG/PNG image.
+
+    Args:
+        image (str): The path to the input GeoTIFF file.
+        output (str): The path to save the output JPEG/PNG file.
+
+    Returns:
+        None
+    """
+
+    import rasterio
+    from PIL import Image
+
+    # Open the GeoTIFF file
+    with rasterio.open(image) as dataset:
```

Comment:
PIL can read a tiff.  It might be simpler to just open the tif with PIL, do any transformations that are necessary, and then image.save(output). I haven't tried that, so I don't know if there are any surprises

Reply:
Good suggestion. Let me look into it.